### ISSUE TITLE
Limit & Offset do not work correctly

### ISSUE THREAD

**Issue by rad-pat** [datafuselabs/databend-sqlalchemy#24]
Small script to replicate limit clause that does not execute against database.
```python
import sqlalchemy as sa
metadata = sa.MetaData()
tbl = sa.Table(
    "test-table",
    metadata,
    sa.Column("Col1", sa.DECIMAL(38, 10)),
    sa.Column("Col2", sa.VARCHAR()),
    sa.Column("Col3", sa.VARCHAR()),
    schema="test-schema",
)
select_query = sa.select(tbl).limit(1000).offset(0)
engine = sa.create_engine("databend://root:@localhost:8000?sslmode=disable")

print(select_query.compile(engine, compile_kwargs={"literal_binds": True, "render_postcompile": True}))
```
Produces this output which will not work
```sql
SELECT "Col1", "Col2", "Col3" 
FROM "test-schema"."test-table"
 LIMIT 0,0
```

Expecting this or similar (something with limit of 1000)
```sql
SELECT "Col1", "Col2", "Col3" 
FROM "test-schema"."test-table"
 LIMIT 1000 OFFSET 0
```

Also relates to https://github.com/datafuselabs/databend-sqlalchemy/issues/26 where parameter substitution does not happen

**Comment by rad-pat** [comment]
Actually, it seems this is not the issue, parameter substitution is not happening for the SQL statement. I will close and log another issue

**Comment by rad-pat** [comment]
Fixed with #27
